src/test/scala/mesosphere/marathon/integration/setup/WithMesosCluster.scala

```diff
@@ -40,7 +40,13 @@ trait WithMesosCluster extends SingleMarathonIntegrationTest { self: Suite =>
   }
 
   def startSlave(name: String, wipe: Boolean = true): Unit = {
-    ProcessKeeper.startMesos(name, s"$mesosWorkDir/$name", Seq("mesos", "slave", "--no-systemd_enable_support", s"--master=zk://${config.zkHostAndPort}/mesos", s"--hostname=$name", s"--work_dir=$mesosWorkDir/$name", s"--port=${processPort(name)}"), "registered with master", wipe)
+    val systemdFlag =
+      if (sys.env.getOrElse("OMIT_SYSTEMD_FLAG", "false") == "true") Seq()
```

Comment:
You could check if not Linux and avoid the env var entirely. Pretty sure we have one of the apache libraries that has this built in, but it's pretty straightforward using only System too.